python/autoencoder/batchmaker.py

```diff
@@ -0,0 +1,72 @@
+import numpy as np
```

Comment:
@exodaniel autoencoders! Nice! :) Would it be easy to move the python folder into the segmatch folder?